code/game/machinery/dance_machine.dm

```diff
@@ -44,26 +54,46 @@
 	song_length = length
 	song_beat = beat
 
+/datum/track/default
```

Comment:
i think you have to overwrite `New()` because won't all this stuff just get overwritten? sorry about the extra work

Reply:
Lol it doesn't even use the code in `New`, it doesn't pass anything... I'm just gonna remove it